webapp/channels/src/components/menu/menu.tsx

```diff
@@ -98,6 +99,7 @@ export function Menu(props: Props) {
     function handleMenuModalClose(modalId: MenuProps['id']) {
         dispatch(closeModal(modalId));
         setAnchorElement(null);
+        props.onMenuModalClose();
```

Comment:
i am curious why did we had to pass this another close function when we already have `dispatch(closeModal(modalId));` on the line above